### ISSUE TITLE
splice annotations

### ISSUE THREAD

**Issue by nicoleruiz** [The-Sequence-Ontology/SO-Ontologies#403]

**Comment by nicoleruiz** [comment]
From Sarah Hunt (seh@ebi.ac.uk):

Matt Hurles group at Sanger have been looking into the impact of variants on splicing and observed:
- that there is some heterogeniety within the canonical splice sites; the donor +5 site is under stronger purifying selection and is more enriched for pathogenic mutations than all other non-canonical splice sites variation at the donor +2 site is typically less damaging than the other 3 canonical splice sites. 

- that the polypyrimidine stretch flanking the acceptor site extends beyond the ‘splice_region’ window and that mutations from pyrimidine to purine are more damaging than other changes.

They requested we annotate these regions more precisely in VEP. We discussed the suggestion on a VATT call and there was broad concensus this would be useful. We already have the term 'splice_donor_5th_base_variant', so would like to propose 2 additional terms:

splice_polypyrimidine_tract_variant

Definition: variant falls in polypyrimidine tract at 3' end of intron, between the 17 and 3 bases from the end 
(acceptor -3 to acceptor -17)
Parent: splicing_variant
splice_donor_region_variant

Definition: variant falls in the region between the 3rd and 6th base after splice junction (5' end of intron)
Parent: splice_site_variant
These are draft suggestions, of course - any improvements would be welcome! It might make sense to make splice_donor_5th_base_variant an child of splice_donor_region_variant, but I don't know if it's too disruptive to introduce new parents.
What do you think?

**Comment by nicoleruiz** [comment]
We have added the following terms:
SO:0002169 splice_polypyrimidine_tract_variant
SO:0002170 splice_donor_region_variant

The definitions are the same as the ones you proposed. We made splice_donor_region_variant a child of splice_region_variant since the definition includes the 3rd and 6th base after the splice junction.

Let us know if these terms and their placement in the ontology work for you.

Nicole
